src/components/AuthTextField/AuthTextField.tsx

```diff
@@ -18,22 +23,45 @@ const AuthTextField: FC<InputProps> = ({
   value,
   onChange,
   placeholder,
+  togglePasswordVisibility = undefined,
+  handleKeyDown = undefined,
+  showPassword = undefined,
+  showEyeIcon = false,
 }: InputProps) => (
   <div className="mb-6 w-full">
     {label && htmlFor && (
       <label htmlFor={htmlFor} className="block text-textColorBlack leading-5 font-medium text-base mb-0.5 font-inter">
         {label}
       </label>
     )}
-    <input
-      type={type}
-      id={id}
-      value={value}
-      onChange={onChange}
-      placeholder={placeholder}
-      required
-      className="w-full bg-white text-base leading-5 border border-secondary rounded-md px-4 py-2 placeholder-textColorBlack opacity-40"
-    />
+    <div className="relative">
+      <input
+        type={type}
+        id={id}
+        value={value}
+        onChange={onChange}
+        placeholder={placeholder}
+        required
+        className={`w-full bg-white text-base text-textColorBlack leading-5 border border-secondary rounded-md  py-2 placeholder-opacity-40 ${
+          showEyeIcon ? 'pl-4 pr-9' : 'px-4'
+        }`}
```

Comment:
Do we have classNames package, then we can improve here a little bit
```js
import classNames from 'classnames';

//...

const inputClassName = classNames('w-full bg-white text-base text-textColorBlack leading-5 border border-secondary rounded-md py-2 placeholder-opacity-40', {
  'pl-4 pr-9': showEyeIcon,
  'px-4': !showEyeIcon
});

//...

<input
  //...
  className={inputClassName}
  //...
/>
```

Reply:
Thank you for the suggestion, I will work on it later.